mods/d2k/rules/starport.yaml

```diff
@@ -3,6 +3,8 @@ mcv.starport:
 	Buildable:
 		Prerequisites: starport
 		Queue: Starport
+	VariedCostMultiplier:
+		Queue: Starport
 	Valued:
 		Cost: 2500
```

Comment:
I missed in the original review, but these custom costs should be removed. It was 75-100% of the original cost in the original game.

Reply:
Done.